### ISSUE TITLE
build coreboot during packaging

### ISSUE THREAD

**Issue by kakaroto** [purism/purism-librem-coreboot-updater#3]
Right now the script downloads the coreboot base file from my server, it should instead clone coreboot and build it so it can include the coreboot_base.rom file in the deb directly instead of relying on a pre-built file hosted somewhere.

**Comment by emanuelb** [comment]
what are the benefits of this? as coreboot is reproducible-built, see
https://tests.reproducible-builds.org/coreboot/coreboot.html 
> reproducible icon PURISM_LIBREM13_V1 (5b161908e02bb9061395ede0cdf959e0df354f8148906d5110b5e2741fa2778e, 8192K) is reproducible.
reproducible icon PURISM_LIBREM13_V2 (56b1110ec5aba88f26b77a064921f7c932f6216d74d12fb3109acdd5c2dba503, 16384K) is reproducible.

it may not be relevant in practice to rebuild if everyone will get the same result, maybe offer both options?
1. pre-built as default
2. re-built if someone wants (thus also records the steps you do to build coreboot)

**Comment by kakaroto** [comment]
I know that coreboot builds are reproducible, this issue isn't about rebuilding the coreboot image on everyone's computer during install, it's about this source deb building coreboot itself and packaging it in the .deb rather than using wget to download it from my own personal server.

**Comment by emanuelb** [comment]
Yeah, it's good :+1: I was confused as content on purism website link to the script as a standalone script without mention that it meant to be a deb package..
